rcl/test/rcl/test_arguments.cpp

```diff
@@ -570,6 +570,63 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_fini_twice) {
   rcl_reset_error();
 }
 
+TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_bad_remove_ros_args) {
+  const char * argv[] = {"process_name"};
+  int argc = sizeof(argv) / sizeof(const char *);
+
+  rcl_allocator_t default_allocator = rcl_get_default_allocator();
+  rcl_arguments_t parsed_args = rcl_get_zero_initialized_arguments();
+  rcl_ret_t ret = rcl_parse_arguments(argc, argv, default_allocator, &parsed_args);
+  ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT({
+    EXPECT_EQ(RCL_RET_OK, rcl_arguments_fini(&parsed_args));
+  });
+
+  const char ** nonros_argv = NULL;
+  int nonros_argc = 0;
+
+  EXPECT_EQ(RCL_RET_INVALID_ARGUMENT, rcl_remove_ros_arguments(
```

Comment:
Is there an error set that should be reset with `rcl_reset_error()`? Same for other invalid returns below.

Reply:
Oops, good catch. Done in https://github.com/ros2/rcl/pull/518/commits/4e5ecfe8f66a9650add4c96155654e9db115d26a